### DISCUSSION TITLE
Directions API?

### DISCUSSION THREAD

**Discussion by hannahaponenko** [transitland/transitland#356] [Q&A] [answered]
If I wanted to go from point A to B and find a list of all available transit options, how would I go about doing that? Would I be able to? It seems that the transitland API parameters don't make this obvious.

**Comment by drewda** [comment, marked as answer]
Hi @hannahaponenko, Transitland APIs do not currently provide directions. To do so requires a separate routing engine. In the future, we may consider making directions/routing available directly through Transitland APIs.

In the meantime, some other options for transit routing:

- proprietary APIs like Google Directions API, HERE, or Bing Maps API
- open-source GraphHopper [self-hosted](https://github.com/graphhopper/graphhopper)
- open-source OpenTripPlanner [self-hosted](https://github.com/opentripplanner/OpenTripPlanner) or [managed hosting by Interline for organizations](https://www.interline.io/opentripplanner/)

**Reply by hannahaponenko** [reply]
Thank you @drewda ! Never thought Bing could be an answer.
